stdlib/runtime/numberUtils.gr

```diff
@@ -895,17 +914,17 @@ export let utoa32 = (value, radix) => {
   } else if (radix == 10n) {
     let decimals = decimalCount32(value)
     let out = allocateString(decimals)
-    utoa32_dec_core(out + 8n, value, decimals)
+    utoa32Buf(out + 8n, value, decimals)
```

Comment:
This is broken because the third argument should be `radix`, but probably better to not change any of these at all so we don't have to compute the number of decimals multiple times.